waydroid/bin/waydroid-mount-shared

```diff
@@ -0,0 +1,16 @@
+#!/bin/pkexec /bin/bash
+# vim: expandtab ft=bash
+
+set -Eeuo pipefail
+
+waydroid_mount() {
+    /usr/bin/bindfs -u droidian -g droidian -p 770 -o nonempty $@
```

Comment:
since we have that already, can we use `$USERNAME` here as well? (perhaps we should retrieve the default group and use that as well instead of hardcoding)